paxexpress_cli/versions/core.py

```diff
@@ -38,6 +61,25 @@ def get_latest(
     return response_handler(response=response, return_model=VersionModel)
 
 
+def get_packages_available_versions(
+    subject: str, repo: Optional[str], package: Optional[str], is_internal: bool = False
+):
+    if not repo:
+        repo = repositories_core.select_from_available_repo(subject=subject)
+    if not package:
+        package = packages_core.select_from_available_packages(
+            subject=subject, repo=repo
+        )
+
+    url = get_url(f"/packages/{subject}/{repo}/{package}/versions/available")
+    response = httpx.get(url=url)
+    if is_internal:
+        return response_handler(
+            response=response, return_with_out_model=True, print_result=False
+        )
+    response_handler(response=response)
```

Comment:
Why no `return` here?

Reply:
we don't need the return value when if `get_packages_available_versions` is called from the command line.